docs/4.0/components/forms.md

```diff
@@ -765,13 +765,37 @@ When attempting to submit, you'll see the `:invalid` and `:valid` styles applied
 // Example starter JavaScript for disabling form submissions if there are invalid fields
 (function() {
   'use strict';
+  // Generic function to manage aria-describedby attribute value on input validity update
+  // @param array of inputs to check
+  // Works only if the invalid-feedback element have an ID
+  function updateAriaDescribedAttribute(inputs) {
+    for (var i = 0; i < inputs.length; i++) {
+      // Check element validity, and set aria-described attribute
+      if(inputs[i].validity.valid === false) {
+        if(inputs[i].parentElement.getElementsByClassName('invalid-feedback').length > 0
+          && inputs[i].parentElement.getElementsByClassName('invalid-feedback')[0].id) {
+          inputs[i].setAttribute('aria-describedby',inputs[i].parentElement.getElementsByClassName('invalid-feedback')[0].id);          
+        }
+      } else {
+        inputs[i].removeAttribute('aria-describedBy');
+      }
+
+      // Listen to input update, and check again the validity
+      inputs[i].addEventListener('input', function() {
```

Comment:
so much event listeners added in case of `input` event triggered, not a good practice in terms of perf :worried: 
this should be done before the first call of `updateAriaDescribedAttribute`
with : 
```javascript
inputs.forEach(function (input) {
  input.addEventListener('input', function () {
    updateAriaDescribedAttribute([this]);
  })
})
```
and no more listeners added when `updateAriaDescribedAttribute` is called